modules/expo-bluesky-gif-view/android/src/main/java/expo/modules/blueskygifview/GifView.kt

```diff
@@ -85,29 +86,27 @@ class GifView(
         .load(source)
         .diskCacheStrategy(DiskCacheStrategy.DATA)
         .skipMemoryCache(false)
-        .listener(
-          object : RequestListener<Drawable> {
-            override fun onResourceReady(
-              resource: Drawable?,
-              model: Any?,
-              target: Target<Drawable>?,
-              dataSource: com.bumptech.glide.load.DataSource?,
-              isFirstResource: Boolean,
-            ): Boolean {
-              if (placeholderRequest != null) {
-                glide.clear(placeholderRequest)
-              }
-              return false
+        .listener(object : RequestListener<Drawable> {
+          override fun onResourceReady(
+            resource: Drawable,
+            model: Any,
+            target: Target<Drawable>?,
+            dataSource: DataSource,
+            isFirstResource: Boolean
+          ): Boolean {
+            placeholderRequest?.let {
+              glide.clear(it)
             }
+            return false
+          }
```

Comment:
No changes here really, but the Glide bump made `Drawable` non-optional for `resource`